bundles/org.openhab.binding.ferroamp/README.md

```diff
@@ -0,0 +1,239 @@
+Data and commands are received/sent using MQTT where the user connects to the MQTT broker residing on the EnergyHub.
+
+Contact Ferroamp support to enable MQTT in the EnergyHub and to get the Username and Password:
+
+https://ferroamp.com/om-ferroamp/
+
+## Supported Things
+
+The binding retrieves data from the different parts of the Ferroamp EnergyHub such as:
+
+- `Ehub`: EnergyHub Wall and EnergyHub XL.
+- `Sso`: Solar string optimizer.
+- `Eso`: Bidirectional DC/DC converter for connection of battery.
+- `Esm`: Energy Storage Module.
+
+## Discovery
+
+Discovery is not supported.
+
+## Thing Configuration
+
+The following configuration-parameters are available.
+
+| Name            | Type    | Description                              | Default | Required | Advanced |
+|-----------------|---------|------------------------------------------|---------|----------|----------|
+| hostName        | text    | Hostname or IP address of the device     | N/A     | yes      | no       |
+| userName        | text    | Username to access the device            | N/A     | yes      | no       |
+| password        | text    | Password to access the device            | N/A     | yes      | no       |
+| hasBattery      | boolean | Is there a battery connected or not?     | N/A     | no       | yes      |
+| ssoS0           | boolean | Is there a first SSO unit connected?     | N/A     | no       | no       |
+| ssoS1           | boolean | Is there a second SSO unit connected?    | N/A     | no       | no       |
+| ssoS2           | boolean | Is there a third SSO unit connected?     | N/A     | no       | no       |
+| ssoS3           | boolean | Is there a fourth SSO unit connected?    | N/A     | no       | no       |
+
+| The unique serial number is marked on the side of the SSO unit. Ex. PS00990-A04-S20120476
+
+## Channels
+
+| Channel Type ID                       | Item Type                  | Read/Write | Label                                    | Description 
+|----------------------------------------------------------------------------------------------------------------------------------------------
+| grid-frequency                        | Number:Frequency           | R          | Estimated Grid Frequency                 | Estimated Grid Frequency |
+| ace-current-l1                        | Number:ElectricCurrent     | R          | ACE Current L1                           | ACE equalization current set-points in Arms |
+| ace-current-l2                        | Number:ElectricCurrent     | R          | ACE Current L2                           | ACE equalization current set-points in Arms |
+| ace-current-l3                        | Number:ElectricCurrent     | R          | ACE Current L3                           | ACE equalization current set-points in Arms |
+| external-voltage-l1                   | Number:ElectricPotential   | R          | External Voltage L1                      | External voltage |
```

Comment:
What is the meaning of external here? what voltage is actually measured? Maybe it is goo enough to adjust the description (also adjust it on the thing-type structure. xml

I see a mix of external, grid and inverter  i suspect that extenral and grid are the same, if so it would be nice to have them aligned.

Reply:
Changed to Grid where it works and total where it works.
About External Voltage, external-voltage-l1 ++ so seems it like they are measured as external ( maybe from some external sensors)

Reply:
20241115
I was wrong, it should be grid-voltage and is now changed

Reply:
fixed.